core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1458,14 +1508,14 @@ private static RelDataType deriveTypeMapConcat(SqlOperatorBinding opBinding) {
   @LibraryOperator(libraries = {SPARK})
   public static final SqlFunction MAP_KEYS =
       SqlBasicFunction.create(SqlKind.MAP_KEYS,
-          ReturnTypes.TO_MAP_KEYS_NULLABLE,
```

Comment:
remove `ReturnTypes.TO_MAP_KEYS_NULLABLE` if we don't use it any more.

Reply:
done, thanks